inc/zoo/swar/SWAR.h

```diff
@@ -196,6 +201,8 @@ constexpr auto isolateLSBits(T v) {
   return v &(lowMask << metaLogFloor(isolateLSB<T>(v)));
```

Comment:
If you use triple slashes, that's a Doxygen comment format, make sure you intend to pass this to Doxygen, otherwise, better don't bother because tools like XCode parse them and would be confused

Reply:
It seemed like you wanted to have comments of this form. Do we not doxygen /autoparse comments?